Add outline color and width for every text element

julio-b opened the pull request:
Define outline colors, i.e. --timeoutlinecolor=FFFF00FF
Define outline width, i.e. --timeoutlinewidth=0.75

New arguments:
```
        --verifoutlinecolor
        --wrongoutlinecolor
        --layoutoutlinecolor
        --timeoutlinecolor
        --dateoutlinecolor
        --greeteroutlinecolor

        --timeoutlinewidth
        --dateoutlinewidth
        --verifoutlinewidth
        --wrongoutlinewidth
        --modifieroutlinewidth
        --layoutoutlinewidth
        --greeteroutlinewidth
```
Closes #147

## Description
 - 

### Screenshots/screencaps
```
./i3lock -n --keylayout 1 --clock \
  --datecolor=ffffffff --dateoutlinecolor=000000ff --datesize=50 --dateoutlinewidth=0.8 \
  --timecolor=000000ff --timeoutlinecolor=ffffffff --timesize=50 --timeoutlinewidth=0.9 \
  --layoutcolor=000000ff --layoutoutlinecolor=aa00aaaa --layoutsize=38 --layoutoutlinewidth=0.8 \
   -B 4
```
![scrot](https://user-images.githubusercontent.com/16379308/105767335-db87d000-5f52-11eb-9c7e-1189b657ae76.png)


## Release notes
Notes: Added outline color and width for every text element

Raymo111 reviewed (changes requested):
I find it wonderful that you were able to make the outline work with almost all the UI elements, but I think the implementation needs a tad more work. In particular, I'd like to keep string lengths to 9, and use new options (i.e. `--timeoutlinecolor` `--timeoutlinewidth`, and so on), just because a color identifier should be max hex + alpha, not `DEADBEEFBADDDCAFE`. Once you use the options just update this PR. Thanks for contributing!

julio-b committed:
Add outline color and width for every text element

Define outline colors, i.e. --timeoutlinecolor=FFFF00FF
Define outline width, i.e. --timeoutlinewidth=0.75

New arguments:
    --verifoutlinecolor
    --wrongoutlinecolor
    --layoutoutlinecolor
    --timeoutlinecolor
    --dateoutlinecolor
    --greeteroutlinecolor

    --timeoutlinewidth
    --dateoutlinewidth
    --verifoutlinewidth
    --wrongoutlinewidth
    --modifieroutlinewidth
    --layoutoutlinewidth
    --greeteroutlinewidth

julio-b commented:
I pushed a second version of the patch.
Changed the arguments according to your suggestions. You are right, they look much better this way.

Raymo111 committed:
Separate the variable definitions

Raymo111 reviewed (approved):
LGTM. Thanks for contributing!
